src/api/uploads/__tests__/FolderUploadNode.test.js

```diff
@@ -116,6 +116,26 @@ describe('api/uploads/FolderUploadNode', () => {
             expect(folderUploadNodeInstance.folderId).toBe(folderId);
         });
 
+        test('should not set error for non-root folders if error is ITEM_NAME_IN_USE', async () => {
+            const folderId = '1';
+            const errorCallback = jest.fn();
+            const isRoot = false;
+            const error = {
+                code: ERROR_CODE_ITEM_NAME_IN_USE,
+                context_info: { conflicts: [{ id: folderId }] },
+            };
+            folderUploadNodeInstance.name = name;
+            folderUploadNodeInstance.createFolder = jest.fn(() => ({
+                id: folderId,
+            }));
+            folderUploadNodeInstance.addFolderToUploadQueue = jest.fn();
```

Comment:
this test does not use this stub to assert anything. we should have an `expect` using this.